src/test/java/seedu/address/storage/CsvManagerTest.java

```diff
@@ -59,6 +66,61 @@ private void exportProperties(ReadOnlyAddressBook addressBook, String filePath)
         exportProperties(addressBook.getPropertyList(), testFolder.resolve(filePath).toFile());
     }
 
+    @Test
+    public void importProperties_nullFile_throwsNullPointerException() {
+        assertThrows(NullPointerException.class, () -> importProperties((File) null));
+    }
+
+    @Test
+    public void importProperties_success() {
+        try {
+            List<Property> expectedProperties = getTypicalAddressBook().getPropertyList();
+            List<Property> properties = importProperties("validProperties.csv");
+            assertEquals(properties, expectedProperties);
+        } catch (ParseException pe) {
+            throw new AssertionError("Import should not fail.", pe);
+        }
+    }
+
+    @Test
+    public void importProperties_invalidFieldFormat_throwsParseException() {
+        assertThrows(ParseException.class,
+                Price.MESSAGE_CONSTRAINTS, () -> importProperties("invalidPropertiesPriceFormat.csv"));
+    }
+
+    @Test
+    public void importProperties_missingColumn_throwsParseException() throws ParseException {
```

Comment:
How valid is it for test cases to throw ParseException? For comparison, no other test cases throw exceptions in the test suite.
An alternative is to catch the ParseException and assert the catch block should not be reached.